src/x86/sse42.rs

```diff
@@ -2,6 +2,7 @@
 use stdsimd_test::assert_instr;
 
 use x86::__m128i;
+use v128::*;
```

Comment:
Please arrange imports in alphabetical order. :-)

Reply:
Good point.